resin/models/device.py

```diff
@@ -60,10 +61,13 @@ def get(self, uuid):
             'eq': uuid
         }
         try:
-            return self.base_request.request(
+            devices = self.base_request.request(
                 'device', 'GET', params=params,
                 endpoint=self.settings.get('pine_endpoint')
-            )['d'][0]
+            )['d']
+            if len(devices) > 1:
```

Comment:
The JS SDK does it because it's doing `startswith: $uuid`, not `eq`, so you really can easily get multiple results. It is unnecessary here, but I'd leave it anyway for now, since we'll want to make this do `startswith` queries too eventually.